apps/wallet-mobile/src/features/Discover/useCases/SearchDappInBrowser/SearchDappInBrowserScreen.tsx

```diff
@@ -9,39 +9,38 @@ import {useBrowser} from '../../common/BrowserProvider'
 import {getGoogleSearchItem} from '../../common/DAppMock'
 import {urlWithProtocol, validUrl} from '../../common/helpers'
 import {useNavigateTo} from '../../common/useNavigateTo'
-import {BrowserSearchToolbar} from '../BrowseDappScreen/BrowserSearchToolbar'
-import {DAppItem} from '../DiscoverListScreen/DAppItem/DAppItem'
+import {BrowserSearchToolbar} from '../BrowseDapp/BrowserSearchToolbar'
+import {DAppItem} from '../SelectDappFromList/DAppItem/DAppItem'
 
 const getUrl = (searchValue: string, isEngineSearch: boolean) => {
   if (isEngineSearch || !validUrl(searchValue)) {
-    const searchUrl = 'https://www.google.com/search?q=' + encodeURIComponent(searchValue)
-    return searchUrl
+    return 'https://www.google.com/search?q=' + encodeURIComponent(searchValue)
   }
 
   return urlWithProtocol(searchValue)
 }
 
-export const BrowseSearchDappScreen = () => {
+export const SearchDappInBrowserScreen = () => {
   const {styles} = useStyles()
-  const router = useRoute<RouteProp<BrowserRoutes, 'browser-search'>>()
+  const router = useRoute<RouteProp<BrowserRoutes, 'discover-search-dapp-in-browser'>>()
   const {isEdit} = router.params ?? {isEdit: false}
   const navigateTo = useNavigateTo()
   const {addBrowserTab, setTabActive, updateTab, tabs, tabActiveIndex} = useBrowser()
   const tabActive = tabs[tabActiveIndex]
-  const [searchValue, setSearchValue] = useState('')
+  const [searchValue, setSearchValue] = React.useState('')
```

Comment:
heads up: we have a hook + provider that transforms the navbar into a searchable bar. (dunno if it's applied on nav bar this one)